pkg/config/env.go

```diff
@@ -134,6 +134,9 @@ var Config = struct {
 	RecorderKafkaKeyFile        string        `env:"FLAGR_RECORDER_KAFKA_KEYFILE" envDefault:""`
 	RecorderKafkaCAFile         string        `env:"FLAGR_RECORDER_KAFKA_CAFILE" envDefault:""`
 	RecorderKafkaVerifySSL      bool          `env:"FLAGR_RECORDER_KAFKA_VERIFYSSL" envDefault:"false"`
+	RecorderKafkaSimpleSSL      bool          `env:"FLAGR_RECORDER_KAFKA_SIMPLE_SSL" envDefault:"false"`
+	RecorderKafkaUsername       string        `env:"FLAGR_RECORDER_KAFKA_USERNAME" envDefault:""`
+	RecorderKafkaPassword       string        `env:"FLAGR_RECORDER_KAFKA_PASSWORD" envDefault:""`
```

Comment:
I think you can add a comment that FLAGR_RECORDER_KAFKA_USERNAME and FLAGR_RECORDER_KAFKA_PASSWORD will only work if FLAGR_RECORDER_KAFKA_SIMPLE_SSL=true

Reply:
Should we add something to the docs or the comment is enough?

Reply:
Hey there, according to [confluent docs](https://docs.confluent.io/current/kafka/authentication_sasl/authentication_sasl_plain.html) you can also use SASL with PLAINTEXT which does not require SSL.

Reply:
> Should we add something to the docs or the comment is enough?

The doc directly shows the content of this env.go file, so the comment here is good enough.

Also the naming of the env, what do you think of `FLAGR_RECORDER_KAFKA_SASL_USERNAME` and `FLAGR_RECORDER_KAFKA_SASL_PASSWORD`? Just to make it clear that it's using SASL